### DISCUSSION TITLE
The 'noun_chunks' syntax iterator is not implemented for language 'nl'. (and potentially other languages)

### DISCUSSION THREAD

**Discussion by davidberenstein1957** [explosion/spaCy#7461] [Help: Coding & Implementations] [answered]
Thanks, the info about multiprocessing vs. not was a helpful clue. This is indeed a bug and the problem is that `noun_chunks` isn't being preserved when the doc is pickled. We will get this fixed in the next patch release of v3.0.

_Originally posted by @adrianeboyd in https://github.com/explosion/spaCy/issues/6891#issuecomment-777590412_

**Comment by davidberenstein1957** [comment]
This issue seems to have only been fixed for the 'en' language models.

**Comment by adrianeboyd** [comment, marked as answer]
Hi, there are two separate issues here:

* Instead of silently returning zero noun chunks, as of v3.0 you get a `NotImplementedError` if noun chunks aren't implemented for a particular language. There's no implementation of `noun_chunks` for Dutch at this point.

   We'd be happy to accept a PR that adds this. You can see how the noun chunks iterator is implemented for different languages by looking at `spacy/lang/*/syntax_iterators.py`. Since the Dutch models use UD trees, you could have a look at languages other than English or German for examples of how noun chunks are can be extracted from UD POS + dependency labels.

* The bug related to #6891 is fixed as of v3.0.2 (for all languages) and isn't related to this issue with Dutch.
